common/main/root.xml

```diff
@@ -2955,9 +2958,18 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			<zone type="Asia/Katmandu">
 				<exemplarCity>Kathmandu</exemplarCity>
 			</zone>
+			<zone type="America/Asuncion">
```

Comment:
Seems quite odd that these moved. Nothing to do with this PR, however.

Reply:
It is because I ran CLDRModify as part of this PR. I guess it had not been done after these zones were added.